### PR TITLE
refactor: change package names to match published packages

### PR DESCRIPTION
**Pull request by amrbashir**
### What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [x] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [x] No

### Checklist
- [x] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [x] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [x] I have added a convincing reason for adding this feature, if necessary

### CODE REVIEW

File: .changes/cli-dev-server-available-port.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-'cli.rs': 'patch'
+'tauri-apps': 'patch'
```
Comment (FabianLars) [inline .changes/cli-dev-server-available-port.md]:
```suggestion
'tauri-cli': 'patch'
```
all the other ones seemed fine

Comment (amrbashir) [reply .changes/cli-dev-server-available-port.md]: nice catch, thanks

**Comment by lucasfernog** [comment]
Why is this needed? @amrbashir

**Comment by amrbashir** [comment]
It was discussed in https://discord.com/channels/616186924390023171/1096095302957682698/1096095302957682698
This is mainly for:
1. consistency
2. improving the github release names to match the npm/cratesio packages
3. make it easier for contributors to guess the name when writing a change file without having to look for it in `.changes/config.json`

**Comment by lucasfernog** [comment]
Ah ok, lots of conflicts to fix :D

**Comment by amrbashir** [comment]
I will update the PR shotly

**Review by lucasfernog — APPROVED** [review]

**Comment by FabianLars** [comment]
@lucasfernog We have to revert this, it just broke the released cli for everyone.

**Pull request merged**